README.md

```diff
@@ -1,8 +1,6 @@
-         HIDAPI library for Windows, Linux, FreeBSD and Mac OS X
-        =========================================================
+# HIDAPI library for Windows, Linux, FreeBSD and Mac OS X
```

Comment:
```suggestion
# HIDAPI library for Windows, Linux, FreeBSD and macOS
```
this should shorten the title and even more accurate right now, according to Apple

Reply:
> This is a plain conversion without any changes to the text.

:) Maybe let's do this as a follow-up change?

Reply:
OK, agree - fix conflict and lets merge